ArduCopter/Parameters.h

```diff
@@ -657,15 +668,15 @@ class ParametersG2 {
 
 #if MODE_ACRO_ENABLED == ENABLED || MODE_SPORT_ENABLED == ENABLED
     // Acro parameters
-    AP_Float                acro_rp_rate;
+    AP_Float                acro_rp_rate;    // remove
```

Comment:
We can actually remove these AP_Float lines (which will free up some memory).  For the enum we normally leave them in but no need to keep these lines.

Reply:
done.  THanks!